### PR TITLE
Updated Cortex XDR IOCs pack names - 20.6.0

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
updated pack name, integration name, and command names.
will also update the pack and integration descriptions.

build currently failing on:
* validate - changing IDs, names, and release notes - should this be 1.0.1?
* market place test - failure is not related to changes (fix already merged to master).

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

### CODE REVIEW

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 1]
```diff
@@ -82,54 +89,70 @@ configuration:
...
-      description: The indicator to enable.
+      description: the indicators
       isArray: true
       name: indicator
-      required: true
+      required: false
+      secret: false
+    deprecated: false
+    description: Push new IOCs to XDR. run This every minute (without indicator argument)
+      or ioc trigerd (using indicator argument).
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:142]:
```suggestion
    description: Push modified IOCs to Cortex XDR.
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 2]
```diff
@@ -82,54 +89,70 @@ configuration:
...
-    execution: false
-    name: xdr-iocs-to-keep
-  - deprecated: false
-    description: Push new IOCs to XDR. This runs every minute.
-    execution: false
-    name: xdr-push-iocs
   - arguments:
     - default: false
-      description: The indicator to enable.
+      description: the indicators
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:135]:
```suggestion
      description:  IOCs to push. leave empty to push all recently modified IOCs.
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 3]
```diff
@@ -82,54 +89,70 @@ configuration:
...
+    description: Push new IOCs to XDR. run This every minute (without indicator argument)
+      or ioc trigerd (using indicator argument).
+    execution: false
+    name: xdr-iocs-push
+  - arguments:
+    - default: false
+      description: In the first run firstTime should be True.
+      isArray: false
+      name: firstTime
+      required: false
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:150]: why do you need this to enable?

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py [hunk 4]
```diff
@@ -332,21 +372,23 @@ def main():
...
+        if command == 'fetch-indicators':
+            fetch_indicators(client, params.get('autoSync', False))
+        elif command in commands:
             commands[command](client)
+        elif command == 'xdr-iocs-sync':
+            xdr_iocs_sync_command(client, demisto.args().get('firstTime', False))
         else:
             raise NotImplementedError(command)
     except Exception as error:
+        pass
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py:391]: remove

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py [hunk 5]
```diff
@@ -322,6 +331,37 @@ def module_test(client: Client):
...
+    else:
+        get_changes(client)
+        if auto_sync:
+            tim_insert_jsons(client)
+            if iocs_to_keep_time():
+                xdr_iocs_sync_command(client)
+
+
+def xdr_iocs_sync_command(client: Client, first_time: bool = False):
+    if first_time:
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py:346]: if the integration context is empty, then you need to run `sync` too, right?
better safe than sorry - don't assume that the actual first call will be with `first_time` true.

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 6]
```diff
@@ -127,20 +148,20 @@ script:
       required: true
       secret: false
     deprecated: false
-    description: Enables IOCs in the XDR server.
+    description: Disables IOCs in the XDR server.
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:151]:
```suggestion
    description: Enables IOCs in the XDR server.
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 7]
```diff
@@ -82,43 +89,58 @@ configuration:
...
+    list, the indicator might still be added to the system.
   display: Bypass exclusion list
   name: feedBypassExclusionList
   required: false
   type: 8
-description: XDR handele indicators
-display: XDR iocs
-name: XDR iocs
+description: Use the Cortex XDR - IOCs feed integration to sync indicators from XSOAR
+  to XDR. Cortex XDR is the world's first detection and response app that natively
```
Comment (glicht) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:109]:
```suggestion
  to Cortex XDR and back. Cortex XDR is the world's first detection and response app that natively
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 8]
```diff
@@ -82,43 +89,58 @@ configuration:
...
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
   display: Bypass exclusion list
   name: feedBypassExclusionList
   required: false
   type: 8
-description: XDR handele indicators
-display: XDR iocs
-name: XDR iocs
+description: Use the Cortex XDR - IOCs feed integration to sync indicators from XSOAR
```
Comment (glicht) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:108]:
```suggestion
description: Use the Cortex XDR - IOCs feed integration to sync indicators from Cortex XSOAR
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/README.md [hunk 9]
```diff
@@ -1,10 +1,12 @@
+Use the Cortex XDR - IOCs feed integration to sync indicators from XSOAR to XDR.
```
Comment (glicht) [inline Packs/XDR_iocs/Integrations/XDR_iocs/README.md:1]:
```suggestion
Use the Cortex XDR - IOCs feed integration to sync indicators from Cortex XSOAR to Cortex XDR and back.
```

**Comment by glicht** [comment]
@yaakovi @esharf see my latest commits. I've updated descriptions and the README.

@kirbles19 please review the changes. We've updated some descriptions and the README.

**Comment by esharf** [comment]
@glicht @yaakovi FYI this should fix the test playbook.
https://github.com/demisto/content-test-conf/pull/851

File: Packs/XDR_iocs/Integrations/XDR_iocs/README.md [hunk 10]
```diff
@@ -1,10 +1,14 @@
+Cortex XDR is the world's first detection and response app that natively integrates network, endpoint and cloud data to stop sophisticated attacks.
 
-XDR handle indicators  
-This integration was integrated and tested with Branch: stable-50 of XDR
-## Configure XDR iocs on Cortex XSOAR  
+Use the Cortex XDR - IOCs feed integration to sync indicators between Cortex XSOAR and Cortex XDR. The integration will sync indicators according to the fetch interval defined. At each interval the integration will push new and modified indicators (selected according to the defined **Sync Query**) from Cortex XSOAR to Cortex XDR. Additionally, the integration will check if there are manual modifications of indicators on Cortex XDR and sync back to Cortex XSOAR. Once daily the integration will perform a *complete sync* which will also remove indicators, which have been deleted/expired in Cortex XSOAR, from Cortex XDR.
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/README.md:3]:
```suggestion
Use the Cortex XDR - IOCs feed integration to sync indicators between Cortex XSOAR and Cortex XDR. The integration will sync indicators according to the defined fetch interval. At each interval, the integration will push new and modified indicators defined in the **Sync Query** from Cortex XSOAR to Cortex XDR. Additionally, the integration will check if there are manual modifications of indicators on Cortex XDR and sync back to Cortex XSOAR. Once per day, the integration will perform a *complete sync* which will also remove indicators that have been deleted/expired in Cortex XSOAR, from Cortex XDR. 
```

File: Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml [hunk 11]
```diff
@@ -20,19 +20,27 @@ configuration:
...
   type: 8
-- display: the severity in XDR
+- defaultvalue: 'true'
+  additionalinfo: When enabled, indicators will be synced from Cortex XSOAR to Cortex XDR. Disable if you wish to use a playbook to sync indicators.
+  display: Auto Sync
+  name: autoSync
+  required: false
+  type: 8
+- additionalinfo: Map the desired severity that will be set to each indicator
+      sent to Cortex XDR.
```
Comment (yaakovi) [inline Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.yml:30]:
```suggestion
- additionalinfo: Map the severity of each indicator that will be synced to Cortex XDR.
```

**Comment by esharf** [comment]
there is a bug in XDR.
the modifying time is not correct some of the time.

**Pull request merged**